lib/bootsy/activerecord/image_gallery.rb

```diff
@@ -10,8 +10,13 @@ module Bootsy
   # that do not point to resources older than the given time
   # limit.
   class ImageGallery < ActiveRecord::Base
-    belongs_to :bootsy_resource, polymorphic: true, autosave: false,
-                                 optional: true
+    if Rails::VERSION::STRING.split(".").first.to_i >= 5
```

Comment:
Thanks for this but we are not maintaining compatibility with Rails <= 4. But FYI in cases like this you can use `Rails::VERSION::MAJOR`.